surface/src/3rdparty/opennurbs/opennurbs_nurbssurface.cpp

```diff
@@ -143,7 +143,7 @@ ON_BOOL32 ON_NurbsSurface::SetDomain(
             )
 {
   bool rc = false;
-  if ( m_order[dir] >= 2 && m_cv_count[dir] >= m_order[dir] && m_knot && t0 < t1 ) {
+  if ( m_order[dir] >= 2 && m_cv_count[dir] >= m_order[dir] && t0 < t1 ) {
```

Comment:
```suggestion
  if ( m_order[dir] >= 2 && m_cv_count[dir] >= m_order[dir] && nullptr != m_knot[dir] && t0 < t1 ) {
```
See https://github.com/mcneel/opennurbs/blob/8.x/opennurbs_nurbssurface.cpp#L149